src/custom-components/EnhancedSelector.tsx

```diff
@@ -0,0 +1,268 @@
+import { SingleDropdownOption, Focusable } from '../deck-components';
+import { Fragment, VFC, useEffect, useMemo, useState } from "react";
+import { BiChevronRight } from "react-icons/bi";
```

Comment:
decky uses Fa for arrow icons iirc

Reply:
all icons have been replaced with fa icons